runners/google-cloud-dataflow-java/arm/build.gradle

```diff
@@ -152,6 +152,7 @@ def cleanUpDockerJavaImages = tasks.register("cleanUpDockerJavaImages") {
       commandLine "gcloud", "--quiet", "container", "images", "untag", "${DockerJavaMultiarchImageName}"
     }
     exec {
+      ignoreExitValue true

```

Comment:
The latest workflow run is failing not because test failure, but failed at cleanup command not successfully executed: https://github.com/apache/beam/actions/runs/6820514264

We do not care about the return code anyways (already did a partial fix in #29124)